sql/ddl_api.sql

```diff
@@ -224,6 +224,34 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION drop_chunks(
```

Comment:
I don't think you need this version. Aren't INTEGERS auto-casted to BIGINT on function calls?